src/sentry/static/sentry/app/components/externalIssues/abstractExternalIssueForm.tsx

```diff
@@ -194,17 +191,23 @@ export default class AbstractExternalIssueForm<
     {trailing: true}
   );
 
+  getDefaultOptions = (field: IssueConfigField) => {
+    const choices = (field.choices as Array<[number | string, number | string]>) || [];
+    return choices.map(([value, label]) => ({value, label}));
+  };
+
   /** If this field is an async select (field.url is not null), add async props. */
   getFieldProps = (field: IssueConfigField) =>
     field.url
       ? {
-          loadOptions: (input: string) => this.getOptions(field, input),
           async: true,
+          autoload: true,
           cache: false,
-          onSelectResetsInput: false,
-          onCloseResetsInput: false,
+          loadOptions: (input: string) => this.getOptions(field, input),
+          defaultOptions: this.getDefaultOptions(field),
           onBlurResetsInput: false,
-          autoload: true,
+          onCloseResetsInput: false,
+          onSelectResetsInput: false,
```

Comment:
@mgaeta I understand that you like making this ordered but I have to say that making things in alphabetical order makes it harder to review PRs because it's hard to know what's actually changing or not. No need to revert since but maybe keep that in mind for the future